tutorials/space/activity3.md

```diff
@@ -158,40 +221,55 @@ sprites.onOverlap(SpriteKind.Player, SpriteKind.Enemy, function (sprite, otherSp
 
 ## Step 7
 
-Inside the ``||sprites:on overlaps||`` block, place a ``||sprites:destroy sprite||`` block. Drag the
-``||variables:otherSprite||`` variable from the ``||sprites:on overlaps||`` block and place it inside
-the  ``||sprites:destroy sprite||``.
+When the enemy collides with the ship, we want it to remove a life then disappear. 
+<hr/>
 
-```blocks
-sprites.onOverlap(SpriteKind.Player, SpriteKind.Enemy, function (sprite, otherSprite) {
-    otherSprite.destroy()
-})
-```
+🔲 Grab the ``||info:change life by [-1]||`` block from the ``||info:Info||`` category and snap it into the 
+``||sprites:on [sprite] of kind [Player] overlaps [othersprite] of kind [Player]||`` container. 
+That will remove one life from the player each time it is hit by an enemy.
 
-## Step 8
+🔲 Find the ``||sprites:destroy [mySprite] ⊕||`` block and snap it below the previous block. 
 
-Now go into ``||info:Info||`` and drag a ``||info:change life by||`` block
-into ``||sprites:on overlaps||``. Now practice dodging the enemies! If you have
-extra time, try experimenting with the ``||scene:camera shake||`` block.
+🔲 To tell the **destroy** block that you want it to affect the overlapping enemy, 
+click on the ``||variables:otherSprite||`` variable from the top of the 
+**overlaps** container and drag it down to replace the 
+``||variables:mySprite||`` argument in ``||sprites:destroy [mySprite] ⊕||``.
+![Grabbing variable from block](/img/space/give-var.gif "So that's how you do that!")
```

Comment:
```suggestion
![Grabbing variable from block](https://raw.githubusercontent.com/microsoft/pxt-skillmap-sample/main/img/space/give-var.gif "So that's how you do that!")
```